### PR TITLE
Testing Project - Lucas B

### PR DESCRIPTION
**Pull request by lucasbeemer**
@bradiowave

### CODE REVIEW

File: basic-javascript/project-1.spec.js [hunk 1]
```diff
@@ -1,3 +1,247 @@
...
+describe('lessThanNinety()', () => {
+    it('should return true if number is less than 90', () => {
+        const expected = true;
+        const actual = helpers.lessThanNinety(89);
+        expect(actual).toEqual(expected);
+    });
+    it('should return false if number is greater than 90', () => {
+        const expected = false;
+        const actual = helpers.lessThanNinety(91);
+        expect(actual).toEqual(expected);
```
Comment (bradencodes) [inline basic-javascript/project-1.spec.js:60]: More specifically, the test should return false if the number is greater than or equal to 90.

File: basic-javascript/project-1.spec.js [hunk 2]
```diff
@@ -1,3 +1,247 @@
...
+    });
+});
+
+//============ SAME LENGTH STRINGS
+describe('areSameLength()', () => {
+    it('should return true if both strings are the same length', () => {
+        const expected = true;
+        const actual = helpers.areSameLength('hello', 'howdy');
+        expect(actual).toEqual(expected);
```
Comment (bradencodes) [inline basic-javascript/project-1.spec.js:35]: Write another test to show that two strings that aren't the same length return `false`.

File: basic-javascript/project-2.spec.js [hunk 3]
```diff
@@ -1,3 +1,199 @@
...
+        expect(actual).toEqual(expected);
+    });
+});
+
+//============ GREETING
+describe('greeting()', () => {
+    it('should return a greeting in chosen language', () => {
+        const expected = 'Hola!';
+        const actual = funcs.greeting('Spanish');
+        expect(actual).toEqual(expected);
```
Comment (bradencodes) [inline basic-javascript/project-2.spec.js:19]: Test the other languages as well.

**Comment by bradencodes** [comment]
## MVP
- [x] Write tests for project-1.js
- [x] Write tests for project-2.js

## Stretch
- [x] Correct bugs and refactor the code
- [x] Write tests for arrays.js
- [ ] Write tests for object.js

## What Looks Great
- MVP complete! Nice work
- Worked on stretch problems as well!
- Frequent and descriptive commits
- Code is well formatted and easy to read
- Comments label each new function test
- Titles for tests are descriptive

## What Could Be Improved
- Tests need to be more thorough and check for edge cases. Also check for data validation and tests that expect to return false.

## Rating (1-3)
**3**
